packages/node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/node",
-  "version": "5.2.10-1",
+  "version": "5.3.0",
```

Comment:
Can you please revert this. You don’t need to update the `version` field in the `package.json`.

Reply:
ok, thanks for fix it